specs/electra/beacon-chain.md

```diff
@@ -270,6 +273,18 @@ class PendingConsolidation(Container):
     target_index: ValidatorIndex
 ```
 
+#### `OnchainAttestation`
+
+*Note*: The container is new in EIP7549.
+
+```python
+class OnchainAttestation(Container):
+    aggregation_bits: Bitlist[MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]
+    data: AttestationData
+    committee_bits: Bitvector[MAX_COMMITTEES_PER_SLOT]
+    signature: BLSSignature
```

Comment:
I think we want to be consistent with the current [`Attestation`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#attestation) by having `committee_bits` to be after the `signature` field